data/human/free worlds war jobs.txt

```diff
@@ -15,6 +15,37 @@ phrase "free worlds scanner payment"
 	word
 		`A Free Worlds captain is waiting for you when you land and takes a copy of your scanner logs, and you collect your payment of <payment>.`
 
+trade
+	commodity "free worlds outfits"
+		"ion engines"
+		"plasma engines"
+		"cooling systems"
+		"fission reactors"
+		"breeder reactors"
+		"thermionic reactors"
+		"battery packs"
+		"fuel cells"
+		"heavy rocket systems"
+		"meteor rocket systems"
+		"javelin rocket systems"
+		"energy blasters"
+		"energy blaster turrets"
+		"torpedo rocket systems"
+		"beam lasers"
+		"beam turrets"
+		"heavy lasers"
+		"heavy laser turrets"
+		"plasma cannons"
+		"plasma turrets"
+		"radar jammers"
+		"tactical scanners"
+		"laser rifles"
+		"fragmentation grenades"
+		"anti-missile turrets"
+		"proton guns"
+		"proton turrets"
+		"shield generators"
```

Comment:
I'm wondering whether it would make more sense to group these into "ship guns", "ship lasers", "ship blasters" etc., because a ship might end up carrying (e.g.) 20 tons of proton turrets (a 34 ton outfit). Or, I could say that the mission is to carry material used in the assembly of proton turrets. It may not be a big deal in the end, but if it can sound better it should be done in my opinion.

Reply:
I think it would be best to say something like `proton turret parts`. I'm not sure how splitting the commodity up makes that better.